### ISSUE TITLE
'make install' endless loop on CentOS 7

### ISSUE THREAD

**Issue by justinclift** [hybridgroup/gocv#282]
On CentOS 7 x64, `make install` goes into an endless loop.

It looks like the Makefile is expecting a (probably newer) version of wget than provided with CentOS:

```
wget: unrecognized option '--show-progress'
Usage: wget [OPTION]... [URL]...

Try `wget --help' for more options.
unzip:  cannot find or open opencv.zip, opencv.zip.zip or opencv.zip.ZIP.
wget: unrecognized option '--show-progress'
Usage: wget [OPTION]... [URL]...

Try `wget --help' for more options.
unzip:  cannot find or open opencv_contrib.zip, opencv_contrib.zip.zip or opencv_contrib.zip.ZIP.
rm: cannot remove ‘opencv.zip’: No such file or directory
rm: cannot remove ‘opencv_contrib.zip’: No such file or directory
```

Removing the `--show-progress` option from the wget line seems to work, although the script bombs out later anyway with `-- Configuring incomplete, errors occurred!`.  (going to look into that shortly)

As a general thought for troubleshooting, it'd probably be useful to have the Makefile detect when something goes wrong so it can skip the automatic `rm -rf` of the temp directory it builds OpenCV in. :wink:

## Your Environment

* Operating System and version:
  * CentOS 7 x64.  Specifically `CentOS Linux release 7.5.1804 (Core)`.
* OpenCV version used:
  * 3.4.2
* How did you install OpenCV?
  * `go get -u -d gocv.io/x/gocv`
* GoCV version used:
  * **3.4.2** is being attempted
* Go version:
  * 1.11
* Did you run the `env.sh` or `env.cmd` script before trying to `go run` or `go build`?
  * No, there's no mention of them in the initial docs.  Should there be?

**Comment by deadprogram** [comment]
Your recent PR was merged into `dev` branch @justinclift I will close this issue once it has been released. Thanks again.

**Comment by deadprogram** [comment]
This has been released as part of v0.16.0 thank you.
